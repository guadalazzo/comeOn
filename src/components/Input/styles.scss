input.input-type {
  font-size: 1rem;
  color: #777 !important;
  line-height: 1.45rem;
  display: block;
  padding: 10px;
  margin-top: 0.33333rem;
  width: 100%;
  border: 0.2rem solid #f5f5f5;
  box-shadow: none;
  outline: none;
  border-radius: 0;
}
input:-webkit-autofill,
input:-webkit-autofill:hover, 
input:-webkit-autofill:focus, 
input:-webkit-autofill:active  {
    -webkit-box-shadow: 0 0 0 30px white inset !important;
}
