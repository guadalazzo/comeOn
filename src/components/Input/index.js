import './styles.scss';
import React from 'react';

const Input = props => <input className="input-type" {...props} />;

export default Input;
