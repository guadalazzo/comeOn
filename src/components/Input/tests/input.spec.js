import React from 'react';
import Input from '../index';
import renderer from 'react-test-renderer';


describe('Test Input component', () => {
    test('Input snapshot', () => {
        const component = renderer.create(<Input/>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
