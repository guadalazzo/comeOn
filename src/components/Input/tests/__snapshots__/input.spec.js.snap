// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Input component Input snapshot 1`] = `
<input
  className="input-type"
/>
`;
